pkg/util/yaml/decoder_test.go

```diff
@@ -125,6 +126,40 @@ stuff: 1
 	}
 }
 
+func TestDecodeBrokenYAML(t *testing.T) {
+	s := NewYAMLOrJSONDecoder(bytes.NewReader([]byte(`---
+	stuff:
+broken-indent: 1
+---       
+  `)), 100)
+	obj := generic{}
+	err := s.Decode(&obj)
+	if err == nil {
+		t.Fatal("expected error with json: prefix, got no error")
+	}
+	if !strings.HasPrefix(err.Error(), "yaml: line 1:") {
```

Comment:
yes, it looks so, but I haven't touched the yaml parser... it seems the bug is in yaml.... I will create follow up issue?